test/jest/integration-capture.spec.ts

```diff
@@ -301,6 +324,67 @@ describe('Integration Capture', () => {
 
             expect(customFlags).toEqual({});
         });
+
+        it('should only return mapped clickIds as custom flags', () => {
+            const integrationCapture = new IntegrationCapture();
+            integrationCapture.clickIds = {
+                fbclid: '67890',
+                _fbp: '54321',
+                _ttp: '0823422223.23234',
+                ttclid: '12345',
+                gclid: '123233.23131',
+                invalidId: '12345',
+            };
+
+            const customFlags = integrationCapture.getClickIdsAsCustomFlags();
+
+            expect(customFlags).toEqual({
+                'Facebook.ClickId': '67890',
+                'Facebook.BrowserId': '54321',
+                'TikTok.ClickId': '12345',
+                'GoogleEnhancedConversions.Gclid': '123233.23131',
+            });
+        });
+    });
+
+    describe('#getClickIdsAsPartnerIdentites', () => {
+        it('should return clickIds as partner identities', () => {
+            const integrationCapture = new IntegrationCapture();
+            integrationCapture.clickIds = {
+                _ttp: '1234123999.123123',
```

Comment:
That is the purpose of the other test "should only return mapped clickIds as partner identities"

Reply:
Yep - Just being extra careful by providing a full list (like in 328) that gets filtered to _just_ the partner identities output, rather than providing it a single key, and having it return that key.  It'd be following the same logic as the test in 328, where that provides a full list of click ids, including a non custom flag click id (_ttP), and returns the others